packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionSmoothers_def.hpp

```diff
@@ -165,7 +203,17 @@ void jacobiIterate(RCP<Teuchos::ParameterList> smootherParams,
                        revisedRowMapPerGrp, rowImportPerGrp); // step 2
 
     for (int j = 0; j < maxRegPerProc; j++) { // step 3
-      regRes[j]->update(1.0, *regB[j], -1.0);
+      regRes[j]->update(SC_ONE, *regB[j], -SC_ONE);
+    }
+
+    // check for convergence
+    {
+      RCP<Vector> compRes = VectorFactory::Build(mapComp, true);
+      regionalToComposite(regRes, compRes, maxRegPerProc, rowMapPerGrp,
+                          rowImportPerGrp, Xpetra::ADD);
+      typename Teuchos::ScalarTraits<Scalar>::magnitudeType normRes = compRes->norm2();
+
+      if (normRes < 1.0e-12) {return;}
```

Comment:
@lucbv Will this conflict with yesterday's removal of the convergence check from the Jacobi smoother in PR #5701?